### DISCUSSION TITLE
EQ3 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#69] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html
Author: @tdegeorge 
Primary Reviewer: @AbbyANoble

**Comment by AbbyANoble** [comment]
I think this is great! Just a few things I noticed:

In [Remark 1.3.3](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#EQ3-3-3) 
it might be good to state the Pythagorean Theorem instead of just referring to it.

Typo in [Definition 1.3.5](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#def-distance-formula): Pythagorean

After [Activity 1.3.6](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#calculating-distance-practice), a remark about the difference between the exact and approximate distance might be helpful.

In [Activity 1.3.8](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#finding-midpoint), could the graph start at the origin instead?

In [Definition 1.3.10](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#def-midpoint-formula) , use display math (the me instead of m) so the formula isn't small.

In [Activity 1.3.11(a)](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#calculating-midpoint-practice-2) 
the correct answer is (-2,-1), but it's not answer choice. I'd also recommend including (4,7) and/or (-4,-7) as decoy answers because that's what you get if you subtract the values instead of add them. (Which I've noticed my students do a lot when they try  to memorize a formula instead of think about average!)

Well now that I just wrote that above, I see that [Activity 1.3.11(b)](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#calculating-midpoint-practice-3) 
is the same question with the correct answer there! So I think you copy/pasted but forgot to change the two points in (a).

In [Activity 1.3.11(c)](https://teambasedinquirylearning.github.io/precalculus/2024/EQ3.html#calculating-midpoint-practice-4) , you haven't given us the coordinates of C so that we can determine B.

**Reply by tdegeorge** [reply]
@AbbyANoble  Fixed all comments except Activity 1.3.8 (now Activity 1.3.9). I originally tried to make it start at the origin, but it is really hard to see the line segment...
